pkg/assetfetcher/http_test.go

```diff
@@ -618,11 +614,17 @@ func TestMakeRequestWithNoAssetsReturned(t *testing.T) {
 }
 
 func TestMakeRequestWithNoResponse(t *testing.T) {
-	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {}))
-	defer ts.Close()
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	mockRT := NewMockRoundTripper(ctrl)
+
+	mockRT.EXPECT().RoundTrip(gomock.Any()).Return(&http.Response{
+		Body: ioutil.NopCloser(bytes.NewReader([]byte("fail"))),
+	}, nil)
+
 	assetFetcher := &NexposeAssetFetcher{
-		Host:     ts.URL,
-		PageSize: 100,
+		HTTPClient: &http.Client{Transport: mockRT},
+		Host:       "http://nexpose-test.com",
```

Comment:
We're mocking the network so we're protected from this for now. However, as a general practice I recommend _always_ using localhost as the target endpoint in tests so that you don't accidentally send traffic to an external system should a future test make use of a non-mocked network. 

You don't need to change anything here. It's more of an FYI for future work.

Reply:
Ok thanks, I changed them all to `localhost`